web/ajax.md

```diff
@@ -104,6 +114,8 @@ This is where the benefit of asynchronous execution comes in. Notice that when t
 - Try the same exercises above using the [`XMLHttpRequest`](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) object instead of AJAX to better understand what the AJAX wrapper does
 
 ### Check for Understanding
-Make the simplest project you can on Codepen.io.  The project must:
+
```

Comment:
Why are you adding line space everywhere ?

Reply:
Actually my text editor automatically formatted,because of plugin I added in my text editor...these changes are not done by me...is it possible to ignore ?

Reply:
Yeah, I think that could be ignored. But it's not a good practice to do so.

Reply:
I think you should revert back them as they were earlier.

Reply:
Sure...From next time onwards I’ll remove that plugin

Reply:
No, fix it this time itself.

Reply:
As the changes are not useful.

Reply:
Okay sure